jck/jck.mk

```diff
@@ -100,14 +100,30 @@ ifndef APPLICATION_OPTIONS
    APPLICATION_OPTIONS :=
 endif
 
+ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
+	APPLICATION_OPTIONS := customJtx=$(Q)
```

Comment:
The original APPLICATION_OPTIONS value will be ignored. If we will handle APPLICATION_OPTIONS in the future PR, please add a TODO comment here.

Reply:
Added todo. Yes, will address this in a separate PR.